src/NServiceBus.AzureStoragePersistence/Subscriptions/Azure/AzureSubscriptionStorage.cs

```diff
@@ -1,100 +1,103 @@
 ﻿namespace NServiceBus.Unicast.Subscriptions
 {
+    using System;
     using System.Collections.Generic;
     using System.Linq;
-    using Azure;
-    using MessageDrivenSubscriptions;
+    using System.Text;
     using Microsoft.WindowsAzure.Storage;
+    using Microsoft.WindowsAzure.Storage.RetryPolicies;
     using Microsoft.WindowsAzure.Storage.Table;
-    using Microsoft.WindowsAzure.Storage.Table.DataServices;
+    using Microsoft.WindowsAzure.Storage.Table.Queryable;
+    using NServiceBus.Azure;
+    using NServiceBus.Unicast.Subscriptions.MessageDrivenSubscriptions;
 
     /// <summary>
-    /// 
     /// </summary>
     public class AzureSubscriptionStorage : ISubscriptionStorage
     {
-        CloudTableClient client;
+        private readonly string subscriptionTableName;
```

Comment:
@SimonCropp good question. This is my first major release with NSB, don't know.
It was public before, but since it's a new major we could internalize it I presume.
@yvesgoeleven any reasons we'd want to keep it (and others) public?